src/aleph_client/asynchronous.py

```diff
@@ -231,7 +231,7 @@ async def create_store(
 
     if magic is None:
         pass
-    elif guess_mime_type is True and "mime_type" not in extra_fields:
+    elif file_content and guess_mime_type is True and ("mime_type" not in extra_fields):
```

Comment:
Why `guess_mime_type is True` and not just `guess_mime_type`?

Reply:
Fixed